source/core/UnitTest.cpp

```diff
@@ -0,0 +1,39 @@
+/**
```

Comment:
Did you consider using one of the existing Unit Test frameworks? GoogleTest is widely used in many teams in NI. As the test code and framework should not be compiled into and thus not really affect the side of the generated vireo.js, it sounds like a good idea to use an existing established framework. Thoughts?